### PR TITLE
feat(footer) Add Stable Longhorn Versions information display

### PR DESCRIPTION
**Pull request by smallteeths**
https://github.com/longhorn/longhorn/issues/3032

image: wangsiye/longhorn-ui:f696a77

Signed-off-by: Siye Wang <siye@rancher.com>

### CODE REVIEW

File: src/components/Layout/Footer.js [hunk 1]
```diff
@@ -80,6 +113,7 @@ function Footer({ app, host, volume, setting, engineimage, eventlog, backingImag
           <a target="blank" onClick={showBundlesModel}>Generate Support Bundle</a>
           <a target="blank" href={issueHref}>File an Issue</a>
           <a target="blank" href="https://slack.cncf.io/">Slack</a>
+          <a target="blank" onClick={showStableLonghornVersions}>{`Stable Longhorn Versions (${stableLonghornVersions})`}</a>
```
Comment (PhanLe1010) [inline src/components/Layout/Footer.js:116]: Can we change the text `Stable Longhorn Versions` to `Newer Stable Versions` ?

**Review by PhanLe1010 — CHANGES_REQUESTED** [review]
We only want to show the stable versions that are newer than the current version. If there is no newer stable version, we don't show the `stable versions` list on the bottom of the page.

**Review by PhanLe1010 — APPROVED** [review]
LGTM. Thanks @smallteeths !

**Review by innobead — APPROVED** [review]
LGTM, but it seems we can't do the filtering of the latest stable versions in the setting page but do in the footer.

But, It's good for now.

**Comment by innobead** [comment]
@smallteeths Please help backport to 1.2.x.

**Comment by PhanLe1010** [comment]
@innobead Can we also backport to v1.1.3 ?

**Comment by innobead** [comment]
> @innobead Can we also backport to v1.1.3 ?

We should. :)

**Pull request merged**